launch/shorty.yml

```diff
@@ -24,7 +24,6 @@ expose:
   health_check:
     type: http
     path: /health/check
-access_scope: org-wide
```

Comment:
Do we no longer need the access_scope for this?

Reply:
Yea we don't. In pods access_scope is deprecated and replaced by `group: org-wide`